### ISSUE TITLE
Image Tiling

### ISSUE THREAD

**Issue by CuongNN218** [ptran1203/traffic_sign_detection#6]
Did image tiling use for training ?

**Comment by ptran1203** [comment]
Yes, I use it for both training and inference. But In training phase, I only randomly crop a region that contains at least one bounding box

you can check it here https://github.com/ptran1203/traffic_sign_detection/blob/main/data_processing.py#L140
